auto_round_extension/vllm_ext/envs_ext.py

```diff
@@ -22,8 +22,10 @@
 # Define extra environment variables
 extra_environment_variables: dict[str, Callable[[], Any]] = {
     "VLLM_MXFP4_PRE_UNPACK_WEIGHTS": lambda: os.getenv("VLLM_MXFP4_PRE_UNPACK_WEIGHTS", "1") in ("1", "true", "True"),
+    "VLLM_MXFP4_PRE_UNPACK_TO_FP8": lambda: os.getenv("VLLM_MXFP4_PRE_UNPACK_TO_FP8", "0") in ("1", "true", "True"),
```

Comment:
if gpu mem is enough, we can unpack the weight to bf16? in current implementation, fp8 weight needs to be dequantized to bf16 during each inference